packages/contract-tests/test/deploy.js

```diff
@@ -1,11 +1,17 @@
-var assert = require("chai").assert;
-var util = require("./util");
+const assert = require("chai").assert;
+const util = require("./util");
 
 describe("Deployments", function () {
-  var Example;
-  var web3;
-  var providerOptions = { vmErrorsOnRPCResponse: false };
-
+  let Example;
+  let web3;
+  // ganache v2 had a default blockGasLimit of 0x6691b7, which these tests rely on
+  const providerOptions = {
+    gasLimit: 0x6691b7,
+    hardfork: "istanbul",
```

Comment:
@davidmurdoch do you know why Istanbul? I think this might have been something you added?

Reply:
istanbul allows running with a `0` gasPrice

Reply:
Huh, no idea why that's something we want here, but I'll take your word for it.  Is it worth adding a comment here saying that in the future this should be updated not to use zero `gasPrice` so this can be moved off Istanbul in the future?

Reply:
Putting this catch-all issue here https://github.com/trufflesuite/truffle/issues/4633